contrib/icinga2-commands.conf

```diff
@@ -8,81 +8,198 @@ template CheckCommand "cloud_aws" {
+        }
+        "-c" = {
+            value = "$cloud_aws_s3_object_crit_bucket_size$"
+            description = "Critical threshold for the size of the specified bucket. Alerts if the size is greater than the critical threshold"
+        }
+        "-w" = {
+             value = "$cloud_aws_s3_object_warn_bucket_size$"
+             description = "Warning threshold for the size of the specified bucket. Alerts if the size is greater than the warning threshold"
+        }
+        "--prefix" = {
+            value = "$cloud_aws_s3_object_prefix$"
+            description = "Limits the response to keys that begin with the specified prefix"
+        }
+        "-p" = {
+            set_if = "$cloud_aws_s3_object_perfdata$"
+            description = "Displays perfdata and lists all objects in the specified bucket"
+        }
+        "--empty-ok" = {
+            set_if = "$cloud_aws_s3_object_empty_ok$"
+            description = "Return OK if there are no buckets"
+        }
+    }
+}
+
+object CheckCommand "cloud_aws_status" {
+    import "cloud_aws"
+
+    command += [ "status" ]
+
+    arguments += {
+        "-u" = {
+            value = "$cloud_aws_status_url$"
+            description = "The AWS Status Page URL"
+        }
+        "-s" = {
+            value = "$cloud_aws_status_service$"
+            description = "The AWS Service to check"
+        }
+    }
+}
+
+apply Service "cloud_aws_single_instance_example" {
     import "generic-service"
 
     check_command = "cloud_aws_instance"
 
-    vars.cloud_aws_name = "IcingaMaster"
+    vars.cloud_aws_ec2_instance_name = "IcingaMaster"
 
     assign where host.name == NodeName
 }
 
-apply Service "cloud_aws_multiple_instances" {
+apply Service "cloud_aws_multiple_instances_example" {
     import "generic-service"
 
     check_command = "cloud_aws_instances"
 
     assign where host.name == NodeName
 }
+
+apply Service "cloud_aws_cloudfront_example" {
+    import "generic-service"
+
+    check_command = "cloud_aws_cloudfront"
+
+    assign where host.name == NodeName
+}
+
+apply Service "cloud_aws_s3_bucket_example" {
+    import "generic-service"
+
+    check_command = "cloud_aws_s3_bucket"
+
+    vars.cloud_aws_s3_bucket_crit_bucket_size = "10G"
+    vars.cloud_aws_s3_bucket_warn_bucket_size = "5G"
+
+    assign where host.name == NodeName
+}
+
+apply Service "cloud_aws_s3_object_example" {
+    import "generic-service"
+
+    check_command = "cloud_aws_s3_object"
+
+    vars.cloud_aws_s3_object_prefix = "file"
+    vars.cloud_aws_s3_object_bucket = [ "foo", "bar" ]
+
+    assign where host.name == NodeName
+}
+
+apply Service "cloud_aws_status_example" {
+    import "generic-service"
+
+    check_command = "cloud_aws_status"
+
+    vars.cloud_aws_status_url = "https://status.aws.amazon.com"
+    vars.cloud_aws_status_service = "cloudwatch"
+
+    assign where host.name == NodeName
+}
```

Comment:
There should be a new line at the end of file.

Reply:
Done